### PR TITLE
Update emmintrin.h

### PR DESCRIPTION
**Pull request by gblikas**
> PR for #22226 

I don't believe this PR is not complete, as I'm still awaiting feedback on the `setr_epi64` methods.

### CODE REVIEW

**Comment by tlively** [comment]
It would be good to add tests for these new functions in test/sse

**Comment by gblikas** [comment]
> It would be good to add tests for these new functions in test/sse

Will do

File: system/include/compat/emmintrin.h [hunk 1]
```diff
@@ -1177,6 +1177,18 @@ _mm_set1_epi8(char __b)
...
+_mm_setr_epi64(__m64 q0, __m64 q1)
+{
+  return (__m128i){ (long long)q0, (long long)q1 };
+}
+
+static __inline__ __m128i __attribute__((__always_inline__, __nodebug__))
+_mm_setr_epi64(int64_t q0, int64_t q1)
+{
+  return (__m128i)wasm_i64x2_make(q0, q1);
+}
```
Comment (sbc100) [inline system/include/compat/emmintrin.h:1190]: This looks like it defines the same function twice.. which I guess means the second one will be ignored?   Should we remove one of them?

Also, can we add some kind of test for this?

Comment (gblikas) [reply system/include/compat/emmintrin.h:1190]: @sbc100 , see https://github.com/emscripten-core/emscripten/pull/22243#issuecomment-2253229600 - in process

**Comment by allsey87** [comment]
@gblikas do you need some help finishing off this PR?

**Pull request closed without merging**